testData/solutions/linux.json

```diff
@@ -1430,15 +1430,53 @@
             "configuration": {
                 "type": "gpii.xrandr",
                 "capabilities": [
-                    "display.screenEnhacements",
+                    "display.screenEnhancement"
                     "applications.org\\.freedesktop\\.xrandr.id"
-                ]
-           }
+                ],
+                "capabilitiesTransformations": {
+                    "screen-resolution": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/screenResolution",
+                            "defaultInputValue": "normal",
+                            "options": {
+                                "high": {
+                                    "outputValue": {
+                                        "width": 1366,
+                                        "height": 768
+                                    }
+                                },
```

Comment:
To allow further transformation code within the outputvalue, whenever it encounters an object, it's interpreted as a transformation statement.. So to simply output the object, this would one option:

```
"outputValue": {
    "transform": {
        "type": "fluid.transforms.literalValue",
        "value": {
            "width": 1366,
            "height": 768
        }
    }
}
```